src/main/thrift/parquet.thrift

```diff
@@ -1255,7 +1259,12 @@ union EncryptionAlgorithm {
  * Description for file metadata
  */
 struct FileMetaData {
-  /** Version of this file **/
+  /** Version of this file 
+    * 
+    * Deprecated.  Readers should determine if they support reading based on
+    * specific metadata (e.g. encoding enum) rather then relying on this field
+    * to make this determination.
+    */
```

Comment:
I disagree with this.  I don't think we should abandon versioning, but rather be more explicit about breaking changes and what is included with version update.  Regardless, this needs more discussion with the community and a clear path forward for how we support breaking changes.

Reply:
I rephrased this to explain rationale.  I agree it needs discussion in the community but I think based on previous conversations this is probably the consensus path.